test/system/700-play.bats

```diff
@@ -1035,3 +1035,72 @@ spec:
     run_podman rm -f -t 0 -a
     run_podman rmi automount_test
 }
+
+@test "podman ply with multi automount volume" {
+    cat >$PODMAN_TMPDIR/Containerfile1 <<EOF
+FROM $IMAGE
+RUN mkdir /test1 /test_same
+RUN touch /test1/a /test1/b /test1/c
+RUN echo "I am from test1 image" > /test_same/hello_world
+VOLUME /test1
+VOLUME /test_same
+EOF
+
+    cat >$PODMAN_TMPDIR/Containerfile2 <<EOF
+FROM $IMAGE
+RUN mkdir /test2 /test_same
+RUN touch /test2/asdf /test2/ejgre /test2/lteghe
+RUN echo "I am from test2 image" > /test_same/hello_world
+VOLUME /test2
+VOLUME /test_same
+EOF
+
+    run_podman build -t test1 -f $PODMAN_TMPDIR/Containerfile1
+    run_podman build -t test2 -f $PODMAN_TMPDIR/Containerfile2
+
+    fname="/$PODMAN_TMPDIR/play_kube_wait_$(random_string 6).yaml"
+    cat >$fname <<EOF
```

Comment:
If at all possible, please see if there's a way to do this using `_write_test_yaml()`

Reply:
Hi Ed,
As the function only support one container in one pod, I just update the first yaml file to use it. But the second one needs two containers, so still create the yaml file directly.